scss/civicrm/contact/pages/_reports.scss

```diff
@@ -1,4 +1,49 @@
-#{civi-page('report-instance')} {
+body[class*='page-civicrm-report-'] {
+
+  #crm-container {
```

Comment:
why is this necessary?

Reply:
Core CiviCRM uses `#crm-container` to style the tables so, instead of adding `!important`s in different css, I chose to take same container as core civicrm to override the style.